src/main/java/org/veupathdb/service/osi/generated/model/HealthResponseStream.java

```diff
@@ -5,8 +5,8 @@
 import java.io.OutputStream;
 import java.util.function.Consumer;
 
-import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.core.StreamingOutput;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.StreamingOutput;
```

Comment:
I assumed the generated files need to be committed, but I was not 100% sure.

Reply:
Yep, we commit them.